File: LeetCode/cpp/771.cpp
class Solution {
public:
    int numJewelsInStones(string J, string S) {
        unordered_set<char> jewel;
        int num = 0;
        for (auto j : J)
            jewel.insert(j);
        for (auto s: S) {
            if (jewel.find(s) != jewel.end())
                ++num;
        }
        return num;
    }
};

File: LeetCode/python3/771.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        num = 0
        jew = set(list(J))
        for s in S:
            if s in jew:
                num += 1
        return num